src/components/Slider/components/SliderCard/components/SliderCardFooter/SliderCardFooter.tsx

```diff
@@ -2,19 +2,31 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { Button, CardActions } from '@mui/material';
 import './SliderCardFooter.scss';
+import { useDispatch } from 'react-redux';
+import { Dispatch } from 'redux';
+import { SliderAnyActions } from '../../../../types/SliderTypes';
+import { sliderSetActiveID } from '../../../../redux/sliderActionsCreators';
 
 interface SliderCardFooterProps {
   id: string;
 }
 
-export const SliderCardFooter = React.memo(({ id }: SliderCardFooterProps) => (
-  <CardActions className="slider-card-footer">
-    <Button size="medium" color="primary" variant="contained">
-      <Link to="#" className="slider-card-read-more-link">
-        Read more
-      </Link>
-    </Button>
-  </CardActions>
-));
+export const SliderCardFooter = React.memo(({ id }: SliderCardFooterProps) => {
+  const dispatch = useDispatch<Dispatch<SliderAnyActions>>();
+
+  return (
+    <CardActions className="slider-card-footer">
+      <Button size="medium" color="primary" variant="contained">
+        <Link
+          to="#"
+          className="slider-card-read-more-link"
+          onClick={e => dispatch(sliderSetActiveID(id))}
```

Comment:
change "e" to more informatively word "event" in handler

Reply:
+